File: 编译文件/1文件/fizzbuzz.exs
mod_pow = fn(base,exp,mod) -> 
:binary.decode_unsigned(:crypto. mod_pow(base,exp,mod))
end

IO.inspect Enum.map 1..100 , &(%{
1 => &1,
6 => "Fizz",
10 => "Buzz",
0 => "FizzBuzz",
}[mod_pow.(&1,4,15)])

File: 编译文件/源码/fizzbuzz.go
package main

import (
  "fmt"
  "math"
)

func main(){
  for n := 1; n<= 100; n++ {
    switch  (int(math.Pow(float64(n),  4.0))%15) {
          case 0:
                fmt.Printf("FizzBuzz\n");
      break;
          case 1:
      fmt.Printf("%d\n",n)
          case 6:
      fmt.Printf("Fizz\n")
      break;
          case 10:
      fmt.Printf("Buzz\n")
      break;      
    }
  }
}